libs/platform/auth/src/oauth/oauth.service.ts

```diff
@@ -115,6 +117,13 @@ export class OauthService implements AuthService, AuthTokenService {
     );
   }
 
+  invokeStoredToken(): void {
+    this.storageService
+      .get<OauthServiceState>(OauthService.STATE_KEY)
+      .pipe(distinctUntilChanged())
+      .subscribe((state) => this.state$.next({ ...state }));
+  }
```

Comment:
Please invoke this via `AppInitializer` feature or in constructor directly instead of manually in the `SwAuthPlugin` to remove unnecessary code.